libs/packages/components/src/lib/external-link/external-link.directive.ts

```diff
@@ -7,43 +7,37 @@ import {
   ComponentFactoryResolver,
   ViewContainerRef,
   OnChanges,
-  HostListener
+  HostListener,
 } from '@angular/core';
-import { FaIconLibrary } from '@fortawesome/angular-fontawesome'; import { fas } from '@fortawesome/free-solid-svg-icons'; import { sds } from '@gsa-sam/sam-styles/src/icons/';
+
 import { FaIconComponent } from '@fortawesome/angular-fontawesome';
 @Directive({
-  selector: 'a[href]'
+  selector: 'a[href]',
 })
 export class ExternalLinkDirective implements OnChanges {
-  private vcRef: ViewContainerRef;
   @Input() href: string;
   @Input() public hideIcon: boolean = false;
+  private internalLinks = ['fsd.gov'];
 
   private get isExternalLink(): boolean {
-    return (
-      this.href
-        .replace(/^https?:\/\//, '')
-        .replace(/^www\./, '')
-        .split('/')[0] != location.hostname
-    );
+    let link = this.href
+      .replace(/^https?:\/\//, '')
+      .replace(/^www\./, '')
+      .split('/')[0];
+    return link != location.hostname && !this.internalLinks.includes(link);
```

Comment:
Instead of `.includes`, could we do equal check? Super edge case scenario here - sd.gov would not evaluate as external link with `.includes`. Non-real links too like d.gov, sd.g, sd.go, etc would also not evaluate as external link